commcare_export/checkpoint.py

```diff
@@ -17,15 +18,25 @@ def set_checkpoint(self, query, query_md5, checkpoint_time=None, run_complete=Fa
             id=uuid.uuid4().hex,
             query_file_name=query,
             query_file_md5=query_md5,
-            time_of_run=checkpoint_time.isoformat(),
+            since_param=checkpoint_time.isoformat(),
+            time_of_run=datetime.datetime.utcnow().isoformat(),
             final=run_complete
         )
         if run_complete:
             self._cleanup(query_md5)
 
     def create_checkpoint_table(self):
-        columns = self._get_checkpoint_table_columns()
-        self._migrate_checkpoint_table(columns)
+        import migrate.versioning.api
+        import migrate.exceptions
+        try:
+            migrate.versioning.api.version_control(self.db_url, self.migrations_repository)
+        except migrate.exceptions.DatabaseAlreadyControlledError:
+            pass
```

Comment:
The `version_control` method creates the 'migrated_version` table in the DB for keeping track of migrations that have been run. If the table already exists then this exception gets thrown.

There's an edge case where a user creates a table called `migrated_version` but then the other operations will fail since the table scheme won't be correct.